src/styles.ts

```diff
@@ -0,0 +1,74 @@
+import { css } from 'lit-element';
+
+export const styles = css`
+
+:host {
+    font-family: var(--font-body);
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    border: 1px solid grey;
+    padding: 12px;
+    border-radius: 12px;
+}
+h2 {
+    color: #414a51;
+    font-size: 24px;
+    text-align: left;
+}
+
+h4 {
+    text-align: center;
+}
+button {
+    font-family: var(--font-body);
+    justify-content: center;
+    margin-top: 20px;
+}
+
+.back-button {
+    padding-bottom: 20px;
+}
+
+input {
+    display: block;
+    border: 1px solid #ccc;
+    outline: none;
+    font-size: 16px;
+    font-family: 'Bariol', sans-serif;
+    font-weight: bold;
+}
+input[type="text"],
+input[type="email"],
+input[type="password"] {
+    font-size: 16px;
+    border-radius: 6px;
+    margin: 6px 0 3px;
+    width: 98%;
```

Comment:
better way to do this is:
`
width: 100%;
padding-left: 2%;
box-sizing: border-box;
`

This will set the width equal to the width including the padding, very handy thing to start using
https://www.w3schools.com/css/css3_box-sizing.asp